src/analysis/seminorm.lean

```diff
@@ -246,6 +246,9 @@ structure seminorm (𝕜 : Type*) (E : Type*) [semi_normed_ring 𝕜] [add_monoi
 (triangle' : ∀ x y : E, to_fun (x + y) ≤ to_fun x + to_fun y)
 
 namespace seminorm
+
+noncomputable theory
```

Comment:
Why did you add this?

Reply:
in my old version I needed it: `definition 'seminorm_sup_finset' is noncomputable, it depends on 'nnreal.canonically_linear_ordered_add_monoid'`

Reply:
If it's no longer needed, can you remove it again?

Reply:
I did that

Reply:
It's still there 😛

Reply:
I removed another instance. this one needs to be there, because `real.has_sup` is non-computable.

Reply:
You can instead just add `noncomputable` on the one or two lines that complain, which might make it clearer to a reader what's computable and what isn't. It's not a big deal though.